manage.py

```diff
@@ -129,12 +129,57 @@ def set_dev_site_name() -> None:
                 name="pythondiscord.local:8000"
             )
 
+    @staticmethod
+    def run_metricity_init() -> None:
+        """
+        Initilise metricity relations and populate with some testing data.
+
+        This is done at run time since other projects, like Python bot,
+        rely on the site initilising it's own db, since they do not have
+        access to the init.sql file to mount a docker-compose volume.
+        """
+        import psycopg2
+        from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+        from urllib.parse import urlsplit
+        # The database URL has already been validated in `wait_for_postgres()`
```

Comment:
This is true, but also fragile since nothing is enforcing that `wait_for_postgres()` is called before this function.

Reply:
Moved to a cls method and standardised here https://github.com/python-discord/site/pull/583/commits/fc3be510115658b5b657dcce37a80f7b60d262ea